frank-doc-frontend/src/app/pipes/as-text.pipe.ts

```diff
@@ -9,33 +9,39 @@ export class AsTextPipe implements PipeTransform {
     if (!value) return "";
     value = value.replaceAll('\\"', '"');
     value = value.replace(/<[^>]*>?/gm, '');
-    value = value.replace(/(?:{@link\s(.*?)})/g, function (match, captureGroup) {
-      // {@link PipeLineSession pipeLineSession}
-      // {@link IPipe#configure()}
-      // {@link #doPipe(Message, PipeLineSession) doPipe}
-      let referencedElement = captureGroup;
-      const hash = captureGroup.indexOf("#");
-      if (hash > -1) {
-        referencedElement = captureGroup.split("#")[0];
+    value = value.replace(/(?:{@link\s(.*?)})/g, (match, captureGroup: string) => {
+      // {@link PipeLineSession pipeLineSession} -> 'PipeLineSession pipeLineSession'
+      // {@link IPipe#configure()} -> 'IPipe#configure()'
+      // {@link #doPipe(Message, PipeLineSession) doPipe} -> '#doPipe(Message, PipeLineSession) doPipe'
```

Comment:
Wat is wat aan de rechter kant van het pijltje staat?

Reply:
De captureGroup result voor het voorbeeld uit de linker kant van de pijl